app/admin/products.rb

```diff
@@ -0,0 +1,148 @@
+include UserUtils
+
+ActiveAdmin.register Product do
+  config.create_another = true
+  actions :all
+
+  # To do: set up index file export
+  index do
+    id_column
+    column 'Product Name', :name
+    column 'Owner Email', :user_email
+    column 'Date Published', :date_published
```

Comment:
Let's also add a column that reflects the `Published` state and uses the out-of-the-box ActiveAdmin thing:
![Screenshot 2024-09-11 at 2 02 02 PM](https://github.com/user-attachments/assets/5e309527-319c-4f90-b19a-3fb4e66db63a)